packages/app/components/drop/drop-form.tsx

```diff
@@ -315,13 +315,22 @@ export const DropForm = () => {
     );
   }
 
-  const handleFileChange = (file: string | File) => {
+  const handleFileChange = (fileObj: FilePickerResolveValue) => {
+    const { file, size } = fileObj;
     let extension;
     // On Native file is a string uri
     if (typeof file === "string") {
       extension = file.split(".").pop();
     }
+    if (size && size > MAX_FILE_SIZE) {
+      setError("file", {
+        type: "custom",
+        message: "File size more than 50MB",
```

Comment:
"File size more than 50MB" -> "Oops, this file is too large (>50MB). Please upload a smaller file."

Reply:
thanks for this copy that's will better!